packages/updatePrescriptionStatus/src/validation/content.ts

```diff
@@ -24,9 +24,11 @@ const COMPLETED_ONLY_BUSINESS_STATUSES = ["collected", "not dispensed", "dispatc
 const IN_PROGRESS_ONLY_BUSINESS_STATUSES = [
   "with pharmacy",
   "with pharmacy - preparing remainder",
-  "ready to collect - partial"
+  "ready to collect",
+  "ready to collect - partial",
+  "ready to dispatch - partial"
 ]
-const AGNOSTIC_BUSINESS_STATUSES = ["ready to dispatch", "ready to dispatch - partial", "ready to collect"]
+const AGNOSTIC_BUSINESS_STATUSES = ["ready to dispatch", "ready to collect"]
```

Comment:
is "ready to collect - partial" meant to be here also?
similar to "ready to dispatch - partial"